app/src/test/java/org/simple/clinic/registration/phone/RegistrationPhoneScreenControllerTest.kt

```diff
@@ -190,54 +188,56 @@ class RegistrationPhoneScreenControllerTest {
   @Test
   fun `when the phone number belongs to an existing user then an ongoing login entry should be created and login PIN entry screen should be opened`() {
     val inputNumber = "1234567890"
-    val mockUser = mock<LoggedInUserPayload>()
-    val userId = UUID.randomUUID()
-    whenever(mockUser.uuid).thenReturn(userId)
-    whenever(mockUser.phoneNumber).thenReturn(inputNumber)
+    val userPayload = PatientMocker.loggedInUserPayload(phone = inputNumber)
 
-    whenever(userSession.findExistingUser(inputNumber)).thenReturn(Single.just(FindUserResult.Found(mockUser)))
-    whenever(userSession.syncFacilityAndSaveUser(any())).thenReturn(Single.just(SaveUserLocallyResult.Success()))
+    whenever(userSession.findExistingUser(inputNumber)).thenReturn(Single.just(FindUserResult.Found(userPayload)))
     whenever(numberValidator.validate(inputNumber, MOBILE)).thenReturn(VALID)
     whenever(userSession.saveOngoingLoginEntry(any())).thenReturn(Completable.complete())
     whenever(userSession.clearOngoingRegistrationEntry()).thenReturn(Completable.complete())
 
     uiEvents.onNext(RegistrationPhoneNumberTextChanged(inputNumber))
     uiEvents.onNext(RegistrationPhoneDoneClicked())
 
-    verify(userSession).saveOngoingLoginEntry(OngoingLoginEntry(uuid = userId, phoneNumber = inputNumber))
-    verify(userSession).syncFacilityAndSaveUser(mockUser)
+    verify(userSession).saveOngoingLoginEntry(OngoingLoginEntry(
+        uuid = userPayload.uuid,
+        phoneNumber = inputNumber,
+        pin = "",
+        fullName = userPayload.fullName,
+        pinDigest = userPayload.pinDigest,
+        registrationFacilityUuid = userPayload.registrationFacilityId,
+        status = userPayload.status,
+        createdAt = userPayload.createdAt,
+        updatedAt = userPayload.updatedAt
+    ))
     verify(userSession).clearOngoingRegistrationEntry()
     verify(screen).openLoginPinEntryScreen()
   }
 
   @Test
-  fun `when the phone number belongs to an existing user and save user locally fails, an error should be shown`() {
+  fun `when the phone number belongs to an existing user and creating ongoing entry fails, an error should be shown`() {
     val inputNumber = "1234567890"
-    val mockUser = mock<LoggedInUserPayload>()
-    val userId = UUID.randomUUID()
-    whenever(mockUser.uuid).thenReturn(userId)
-    whenever(mockUser.phoneNumber).thenReturn(inputNumber)
-
-    whenever(userSession.findExistingUser(inputNumber)).thenReturn(Single.just(FindUserResult.Found(mockUser)))
-
-    whenever(userSession.syncFacilityAndSaveUser(any())).thenReturn(
-        Single.just(SaveUserLocallyResult.NetworkError()),
-        Single.just(SaveUserLocallyResult.UnexpectedError())
-    )
+    val userPayload = PatientMocker.loggedInUserPayload(phone = inputNumber)
```

Comment:
TODO: Add a `TODO` here to revisit the validity of this test because a network call (sync facilities) is no longer being made and a crash here indicates a file storage problem which is hard to recover from.